src/pages/dmt5/_dhero.js

```diff
@@ -0,0 +1,302 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import { deriv_app_url } from 'common/utility'
+import { localize } from 'components/localization'
+import { Flex } from 'components/containers'
+import { Header, QueryImage } from 'components/elements'
+import { LinkButton } from 'components/form'
+import device from 'themes/device.js'
+
+const Wrapper = styled(Flex)`
+    position: relative;
+    justify-content: flex-start;
+    background-color: var(--color-black);
+    height: 60.2rem;
+    padding: 12rem 12rem 9rem 12rem;
+
+    @media (max-width: 1254px) {
+        padding: 8rem 12rem 9rem 4rem;
+    }
+    @media ${device.laptopM} {
+        height: 52.7rem;
```

Comment:
for media query we can use pixel for more consistent sizing 🙏

Reply:
Sure Amam, but in this PR if I wanna change them to `px`, I have to check the whole pages from the first, because the numbers will be different with `rem`. Actually, I have done this before for all `mobileL` breakpoints, but for bigger sizes I have to crate a card and convert all of `rem`s to `px`.